src/http-client.js

```diff
@@ -3,13 +3,14 @@ import {Headers} from './headers';
 import {RequestBuilder} from './request-builder';
 import {HttpRequestMessage,createHttpRequestMessageProcessor} from './http-request-message';
 import {JSONPRequestMessage,createJSONPRequestMessageProcessor} from './jsonp-request-message';
+import { RequestMessageProcessor } from './request-message-processor';
```

Comment:
Other imports in this project use no space around the curly braces, and it'd be nice to remain consistent.